src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/SpaceManagementTest.java

```diff
@@ -120,27 +121,33 @@ void setupKernel(Path storeDirectory)
     @Test
     void GIVEN_user_component_config_with_space_management_WHEN_space_exceeds_THEN_excess_log_files_are_deleted()
             throws Exception {
-        tempDirectoryPath = Files.createDirectory(tempRootDir.resolve("IntegrationTestsTemporaryLogFiles"));
+        // Given
 
+        tempDirectoryPath = Files.createDirectory(tempRootDir.resolve("IntegrationTestsTemporaryLogFiles"));
+        // This method configures the LogManager to get logs with the pattern ^integTestRandomLogFiles.log\w* inside
+        // then tempDirectoryPath with a diskSpaceLimit of 105kb
         setupKernel(tempDirectoryPath);
-        TimeUnit.SECONDS.sleep(10);
 
-        CountDownLatch cdl = new CountDownLatch(5);
-        Consumer<GreengrassLogMessage> listener = (m) -> {
-            if (m.getMessage().contains("Successfully deleted file")) {
-                cdl.countDown();
-            }
-        };
-        try (AutoCloseable l = createCloseableLogListener(listener)) {
-            for (int i = 0; i < 15; i++) {
-                createTempFileAndWriteData(tempDirectoryPath, "integTestRandomLogFiles.log_", "");
-            }
-            //Todo: we should not be checking for log messages to verify if a log file was deleted.
-            assertTrue(cdl.await(60, TimeUnit.SECONDS), "5 files deleted");
+        // When
+
+        // The total size will be 150kb (each log file written is 10kb * 15) given the max space is 105 kb, then we
+        // expect it to delete 5 files for the total log size to be under 105kb
+        for (int i = 0; i < 15; i++) {
+            createTempFileAndWriteData(tempDirectoryPath, "integTestRandomLogFiles.log_", "");
         }
 
         Pattern logFileNamePattern = Pattern.compile("^integTestRandomLogFiles.log\\w*");
         LogFileGroup logFileGroup = LogFileGroup.create(logFileNamePattern, tempDirectoryPath.toUri(), mockInstant);
-        assertEquals(10, logFileGroup.getLogFiles().size());
+        // Then
+
+        assertThat("Log group size not below 105 kb",() -> {
```

Comment:
"log group size should eventually be less than 105 KB" - you're checking for <=105 to be eventually true, so I don't see how this text is currently accurate.